### DISCUSSION TITLE
Appeal regarding the refusal of access to the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by theo-pezet** [community/community#141998] [GitHub Education]
### Hi everyone,

Hello,

I am contacting you regarding my application for the GitHub Student Developer Pack, submitted on October 19, 2024, under the account @theo-pezet. Unfortunately, I received a message stating that my request could not be approved because my account was created too recently.

I would like to point out that I have provided all the required information, including a certificate of enrollment to prove my student status. The rejection appears to be based solely on the recent creation of my account, and I was hoping if it would be possible to reconsider my application, taking these elements into account.

I actively use GitHub for academic purposes, and access to the Student Developer Pack would be immensely helpful for my projects. Please let me know if you need any additional information to complete the verification process.

Thank you in advance for your assistance and understanding.

Best regards,
Theo Pezet (@theo-pezet)

**Comment by VictorMoni** [comment]
Hi @theo-pezet 😄👋,  

Thanks for sharing your situation. Unfortunately, GitHub usually requires an account to be a bit more established before approving the Student Developer Pack, even if all documents are correct.  

My advice would be:  
- Keep your account active (commits, projects, contributions) for a few weeks.  
- Then reapply with the same enrollment certificate.  
- If needed, you can also open a support ticket at [GitHub Education Support](https://support.github.com/) to clarify your case directly.  

Hope this helps, and good luck with your application! 🚀
